benchmarking/evaluations/scenarios/multiple_tokenization_goals.py

```diff
@@ -0,0 +1,53 @@
+from typing import List
+
+from api.models.enums import (
+    DiversifyType,
+    ScenarioAttribute,
+    TokenizationConstraintDirection,
+)
+from api.models.tokenization_constraint import TokenizationConstraint
+from benchmarking.evaluations.goals import DiversityGoal, WeightGoal
+from benchmarking.evaluations.interfaces import Scenario, Goal
+
+
+class MultipleTokenizationGoals(Scenario):
+    def __init__(self, value_of_gpa: int, value_of_race: int, value_of_age):
+        self.value_of_gpa = value_of_gpa
+        self.value_of_race = value_of_race
+        self.value_of_age = value_of_age
+
+    def name(self) -> str:
+        return "Scenario to run multiple tokenization goals"
+
+    @property
+    def goals(self) -> List[Goal]:
+        return [
+            DiversityGoal(
+                DiversifyType.CONCENTRATE,
+                ScenarioAttribute.GPA.value,
+                tokenization_constraint=TokenizationConstraint(
+                    direction=TokenizationConstraintDirection.MAX_OF,
+                    threshold=3,
+                    value=self.value_of_gpa,
+                ),
+            ),
+            DiversityGoal(
+                DiversifyType.DIVERSIFY,
+                ScenarioAttribute.RACE.value,
+                tokenization_constraint=TokenizationConstraint(
+                    direction=TokenizationConstraintDirection.MIN_OF,
+                    threshold=2,
```

Comment:
why is this threshold different?

Reply:
I think this was just because I basically just wanted a scenario with more varied parameters to see how the algorithm performed, but I was not sure what exactly to call it. I will come up with a better name and I think that will solve most of the problems in this PR.